src/ophyd_async/core/soft_signal_backend.py

```diff
@@ -109,14 +142,20 @@ class SoftSignalBackend(SignalBackend[T]):
     _initial_value: Optional[T]
     _timestamp: float
     _severity: int
+    _units: Optional[str]
+    _precision: Optional[int]
 
     def __init__(
         self,
         datatype: Optional[Type[T]],
         initial_value: Optional[T] = None,
+        units: Optional[str] = None,
+        precision: Optional[int] = None,
```

Comment:
Or SoftSignalBackend can take kwargs of other fields to pass as configuration.
e.g. units, precision, choices, limits